qutip/core/operators.py

```diff
@@ -50,9 +52,10 @@
 from . import data as _data
 from .qobj import Qobj
 from .dimensions import flatten
+from .data import to
```

Comment:
Personally I'd like to keep "QuTiP house style" to be `from . import data as _data`, and then use `_data.to` exclusively to be hyper clear what's being converted, but open to other views.